lldb/test/API/lang/swift/designated_initializer_self/main.swift

```diff
@@ -3,6 +3,7 @@ class C {
   convenience init(unused: Bool) { 
     self.init()
     print(1)//%self.expect('po self', substrs=['<C: 0x'])
+            //%self.expect('v self', substrs=['(Self) self = 0x'])
```

Comment:
I feel like we shouldn't want this to show as `Self`.

Reply:
Self is the type as it appears in the source code. Maube the static type should be Self and the dynamic type should be the actual type?